<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetectionSettingsDialog</class>
 <widget class="QDialog" name="DetectionSettingsDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>430</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Motion Detection Parameters</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Blur Radius</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Blur Std Dev</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Background Blend Threshold</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Foreground Detect Threshold</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Background Threshold Timeout (Frames)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Entity Timeout (Frames)</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Detection Timeout (Frames)</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Background Blend Ratio</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Foreground Blend Ratio</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Detection Dilation Factor</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Blending Dilation Factor</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Border Width (Pixels)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_blurRadius"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_blurStdev"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_blendingThreshold"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_detectionThreshold"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_thresholdTimeout"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_entityTimeout"/>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEdit_detectionTimeout"/>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEdit_BackgroundBlendRatio"/>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="lineEdit_ForegroundBlendRatio"/>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="lineEdit_ForegroundOverloadRatio"/>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="lineEdit_dilateDetectionFactor"/>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="lineEdit_dilateBlendingFactor"/>
    </item>
    <item row="12" column="1">
     <widget class="QLineEdit" name="lineEdit_borderWidth"/>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Foreground Overload Percentage</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QCheckBox" name="checkBox_Greyscale">
      <property name="text">
       <string>Enabled</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Greyscale Processing (Stop/start to apply)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DetectionSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DetectionSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>DetectionSettingsDialog</receiver>
   <slot>handleButton(QAbstractButton*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>handleButton(QAbstractButton*)</slot>
 </slots>
</ui>
